javascript/nodejs/eval_node.yaml

```diff
@@ -0,0 +1,113 @@
+rules:
+  - id: eval_nodejs
+    patterns:
+      - pattern-inside: |
+          $APP.$METHOD(..., function $FUNC($REQ, $RES, ...){ ... });
```

Comment:
You might be able to improve the precision of this rule with 

```
- pattern-inside: |
      const $EXPRESS = require('express');
      ...
      const $APP = $EXPRESS();
      ...
      $APP.$METHOD(..., function $FUNC($REQ, $RES, ...){ ... });
```

Reply:
@DrewDennison Why is this matching the eval outside?

https://semgrep.live/oQx?version=develop

Reply:
Seems like a bug. Sometimes `pattern-inside` will stretch beyond its boundaries trying to resolve `...`. Your initial pattern-inside, `$APP.$METHOD(..., function $FUNC($REQ, $RES, ...){ ... });` is probably sufficient for now!

Tracking here: https://github.com/returntocorp/semgrep/issues/708